database/tables/pl_courses.pg

```diff
@@ -9,7 +9,10 @@ columns
     institution_id: bigint not null default 1
     options: jsonb not null default '{}'::jsonb
     path: text
+    question_sharing_enabled: boolean default false
```

Comment:
We should probably use the new feature flags system instead of bespoke feature-specific columns.

Reply:
Sounds good, I'll work on it.

Reply:
Done!